constants/network.h

```diff
@@ -0,0 +1,10 @@
+#ifndef CONSTANTS_H
```

Comment:
(discussion) I'll have to experiment with cmake // make but at work we use `#pragma once` and let the compiler do the include guards automatically. I think it's cleaner, I'll look into this when I look at cmake